.travis.yml

```diff
@@ -87,6 +87,11 @@ jobs:
               - pip install -e .[funsor]
               - pytest -vs --cov=pyro --cov-config .coveragerc --stage funsor --durations 10
               - CI=1 pytest -vs --cov=pyro --cov-config .coveragerc --stage test_examples --durations 10 -k funsor
+        - name: type_check
```

Comment:
nit: could you move this to just below the docs stage so it is run before the unit test stage?

Reply:
Sure, will do that.